ansible/tasks/setup-postgres.yml

```diff
@@ -239,7 +239,7 @@
   become_user: postgres
   shell: |
     source /var/lib/postgresql/.bashrc
-    /usr/lib/postgresql/bin/pg_ctl -D /var/lib/postgresql/data start
+    /usr/lib/postgresql/bin/pg_ctl -D /var/lib/postgresql/data start -o "-c shared_preload_libraries='pg_net'"
```

Comment:
Not sure why this is needed? Isn't pg_net already included?

Reply:
> psql:/tmp/migrations/db/migrations/20240822021428_enable_webhooks_by_default.sql:4: ERROR:  pg_net is not in shared_preload_libraries

It's not included in this code path for reasons unclear to me https://github.com/supabase/postgres/actions/runs/10495775396/job/29074946651

Reply:
Are we happy to merge this one?